web-local/src/lib/components/workspace/explore/filters/Filters.svelte

```diff
@@ -179,19 +182,19 @@ The main feature-set component for dashboard filters
   </div>
   {#if currentDimensionFilters?.length}
     <ChipContainer>
-      {#each currentDimensionFilters as { name, sqlName, dimensionId, selectedValues, filterType } (dimensionId)}
+      {#each currentDimensionFilters as { name, label, sqlName, selectedValues, filterType } (name)}
         {@const isInclude = filterType === "include"}
         <div animate:flip={{ duration: 200 }}>
           <RemovableListChip
-            on:toggle={() => togglerFilterMode(dimensionId)}
+            on:toggle={() => togglerFilterMode(name)}
             on:remove={() =>
-              clearFilterForDimension(dimensionId, isInclude ? true : false)}
-            on:apply={(event) => toggleDimensionValue(event, { dimensionId })}
+              clearFilterForDimension(name, isInclude ? true : false)}
+            on:apply={(event) => toggleDimensionValue(event, { name })}
             on:search={(event) => {
-              setActiveDimension(sqlName, dimensionId, event.detail);
+              setActiveDimension(sqlName, name, event.detail);
```

Comment:
we should get rid of `activeDimensionId`. It's same as `name`. Also `sqlName` would be same as `name`